src/runtime/propensity-server-test.js

```diff
@@ -296,4 +312,132 @@ describes.realWin('PropensityServer', {}, env => {
           expect(() => {throw reason;}).to.throw(/Invalid request/);
         });
   });
+
+  it('should listen for events from event manager', function*() {
+    //event registration happens during initialization so we need to stub it
+    //and then create a new propensity server
+    const eventMan = new ClientEventManager();
```

Comment:
Why not use eventManager object created in line 43?

Reply:
The comment above it explains why I had done it.  I've actually changed the implementation so it might not be necessary now - it depends how I register the listener.  

The syntax now: eventManager.registerEventListener(event => this.handleClientEvent_(event));
The way I had registered it initially: eventManager.registerEventListener(this.handleClientEvent_.bind(this));

For the second implementation, if I had stubbed it after the registration occurred then it would remember the original version of the function rather than the stubbed version (so it didn't work).  The way I'm doing it now doesn't matter.  I can do it either way.  I'm not sure which method is preferable.